internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -63,18 +61,26 @@ func createMountsConfig(ctx context.Context, coi *createOptionsInternal) (*mount
 					return nil, fmt.Errorf("failed to eval symlinks for mount source %q: %s", mount.Source, err)
 				}
 				mdv2.HostPath = src
+			} else if mount.Type == "virtual-disk" || mount.Type == "physical-disk" || mount.Type == "ExtensibleVirtualDisk" {
```

Comment:
extensible-virtualdisk or ExtensibleVirtualDisk?

Reply:
I believe he'll be changing this. It was originally `ExtensibleVirtualDisk` in his cri PR https://github.com/jterry75/cri/pull/97, but I'd said maybe we should follow the same format we use for the others of `lowercaseword1-lowercaseword2`

Reply:
So in the container config we use the mount type as `extensible-virtual-disk` so that we can stay consistent with the naming scheme of other mounts (i.e `virtual-disk`, `physical-disk` etc) but when actually sending the request to the HCS it needs to be `ExtensibleVirtualDisk`. However here I missed to replace this with `extensible-virtual-disk`.

Reply:
Sounds good. Note there is one more place where this change is needed in the PR as well.